dashboards/Insight/Prometheus_Exporter_Status.json

```diff
@@ -4610,10 +4727,13 @@
         "allFormat": "glob",
         "current": {
           "selected": false,
-          "text": "ip-10-178-0-240.us-east-2.compute.internal",
-          "value": "ip-10-178-0-240.us-east-2.compute.internal"
+          "text": "mongo",
+          "value": "mongo"
```

Comment:
Can you change to "text": "",
                               "value": "" ?

Reply:
Done